server/services/client.ts

```diff
@@ -391,4 +395,60 @@ export = ({ strapi }: StrapiContext): IServiceClient => ({
       status
     );
   },
+
+
+  async sendResponseNotification(
+    this: IServiceClient,
+    entity: Comment
+  ): Promise<void> {
+
+    if (entity.threadOf) {
+      const thread = isObject(entity.threadOf) ? entity.threadOf : await this.getCommonService().findOne({ id: entity.threadOf });
+      let emailToSend = thread?.author?.email;
+      if (thread.authorUser && !emailToSend) {
+        const strapiUser = isObject(thread.authorUser) ? thread.authorUser : await strapi.db.query<StrapiUser>("api::user").findOne({ 
+          where: { id: thread.authorUser }
+        });
+        emailToSend = strapiUser?.email;
+      }
+
+      if (emailToSend) {
+        const superAdmin = await strapi.db
```

Comment:
Do we need to reference db connection? Isn't `strapi.query` sufficient?

Reply:
it might be used by both ways. In this plugin I'm consistent is `strapi.db`